atbuild/src/main.swift

```diff
@@ -27,16 +27,22 @@ import Glibc
 let defaultBuildFile = "build.atpkg"
 
 func loadPackageFile() -> Package {
-
-    //build overlays
-    var overlays : [String] = []
+    var overrides: ConfigMap? = nil
+    
+    var overlays: [Value] = []
     for (i, x) in Process.arguments.enumerate() {
-        if x == "--overlay" {
+        if x == "--use-overlay" {
```

Comment:
We need to update the documentation at https://github.com/AnarchyTools/atbuild/blob/master/docs/overlays.md